### DISCUSSION TITLE
Used valid Wifi SSID still in use when changed to an not available SSID

### DISCUSSION THREAD

**Discussion by digixx** [micropython/micropython#11472] [RP2040 / Pico]
I use a Raspberry Pico W with a sensor to send some data to my MQTT server over Wifi. The development IDE is thonny. When I set the SSID on the main.py to an available Wifi-SSID, the connection is made and data can be send. If I stop the program and set in main.py the SSID to a value does not exist, save it and run it again, the connection is still working. If I power cycle the Pico, no connecction can be made.

It seems that software reboot does not clear Wifi settings correctly.

**Comment by scruss** [comment]
Yes, that's typically correct. The wifi subsystem is a completely different processor, and isn't reset on soft reset
